### PR TITLE
DB backup script

### PR DESCRIPTION
**Pull request by sirjoe**

### CODE REVIEW

File: backup/efcRosterQa_20180214.sql [hunk 1]
```diff
@@ -0,0 +1,196 @@
+-- MySQL dump 10.13  Distrib 5.6.38, for macos10.12 (x86_64)
```
Comment (josephj) [inline backup/efcRosterQa_20180214.sql]: Do we need to commit this file?

Comment (sirjoe) [reply backup/efcRosterQa_20180214.sql]: no, should have updated `.gitignore`. One sec

File: server.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ process.env.TZ = 'Australia/Sydney';
 require('./newrelic');
```
Comment (josephj) [inline server.js]: Maybe add `/*global Cronjob */` to the first line if you can't import this Cronjob module.

Comment (sirjoe) [reply server.js]: It was a typo before lol. Should've been `CronJob` instead of `Cronjob`.  Joseph 🔫

**Pull request merged**